src/syscheckd/src/db/include/db.hpp

```diff
@@ -106,12 +106,13 @@ class EXPORTED DB final
     void getFile(const std::string& path, std::function<void(const nlohmann::json&)> callback);
 
     /**
-    * @brief countFiles Count files in the database.
+    * @brief countEntries Count files in the database.
     *
+    * @param tableName Table name.
     * @param selectType Type of count.
     * @return Number of files.
     */
-    int countFiles(const COUNT_SELECT_TYPE selectType);
+    int countEntries(const std::string& tableName, const COUNT_SELECT_TYPE selectType);
```

Comment:
This function could be called countRegistries for better visualization

Reply:
This function work to count both file and registries